packages/main/src/plugin/updater.ts

```diff
@@ -62,6 +64,22 @@ export class Updater {
     this.#updateCheckResult = undefined;
   }
 
+  public async openReleaseNotes(version: string): Promise<void> {
+    if (version === 'current') {
+      version = app.getVersion();
+    } else if (version === 'latest') {
+      version = this.#nextVersion?.substring(1) ?? '';
+    }
+    const urlVersionFormat = version.split('.', 2).join('.');
+    let notesURL = `https://podman-desktop.io/blog/podman-desktop-release-${urlVersionFormat}`;
+    https.get(notesURL, res => {
+      if (res.statusCode !== 200) {
+        notesURL = `https://github.com/containers/podman-desktop/releases/tag/v${version}`;
```

Comment:
IMHO it should rely on the `repository` field of the package.json file

Reply:
for example the pre-releases are in another repository and that field will be accurate